.github/workflows/ci.yml

```diff
@@ -28,6 +28,9 @@ jobs:
            # Install specific podman version
            sudo apt-get upgrade
 
+      - name: Build a container for CPU inferencing
+        run: ./container_build.sh build ramalama
```

Comment:
Should these be build_rm?

Reply:
In this case no, because I want the ci.yml to try and use this image

Otherwise we don't find issues like "llama-run library not found", because it will use the one from quay.io

Reply:
This probably isn't complete yet here, but I want to force our test suite to not pull down an image, instead use a locally built one

Reply:
SGTM